### PR TITLE
Example PR workflow

### PR DESCRIPTION
**Pull request by Shaomei**
Updated the README

### CODE REVIEW

**Comment by Shaomei** [comment]
@blickly can you review the PR for me?

File: app/home.js
Comment (blickly) [inline app/home.js:1]: I don't think you meant to add this file. Can you revert it in this PR?

File: package-lock.json
Comment (blickly) [inline package-lock.json:1]: Probably this one too.

**Review by blickly — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (Shaomei) [reply package-lock.json:1]: Done

File: app/home.js
Comment (Shaomei) [reply app/home.js:1]: Done.

**Comment by blickly** [comment]
LGTM

**Pull request merged**